virttest/libvirt_remote.py

```diff
@@ -44,3 +44,20 @@ def update_remote_file(params, value,
     except Exception as err:
         raise exceptions.TestFail("Unable to update {}: {}"
                                   .format(file_path, err))
+
+
+def delete_remote_file(file, params):
```

Comment:
is it possible to wrap file into params since the latter already includes many parameters?

Reply:
Actually, I prefer to keep this way, because it's easy to find out the file path here instead of checking or modifying dictionary.